app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> None:
+        if "vaccine" not in visitor.keys():
+            raise NotVaccinatedError("NotVaccinatedError in line 13")
```

Comment:
this error message is not good enough, why do you need the line number and what's the point of the error message to be the same as the error name? fix everywhere